### PR TITLE
LOG-1068: Refactor operator code to not rely explicitly on openshift-logging

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR preps the CLO to make deployment namespace configurable

### Links
 https://issues.redhat.com/browse/LOG-1068

/hold

### CODE REVIEW

**Comment by jcantrill** [comment]
blocked by https://github.com/openshift/cluster-logging-operator/pull/1716

File: hack/testing-olm/test-010-deploy-via-olm-minimal.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@ set -eou pipefail
 source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
 source "$(dirname $0)/assertions"
 
+LOGGING_NS=${LOGGING_NS:-"openshift-logging"}
```
Comment (vparfonov) [inline hack/testing-olm/test-010-deploy-via-olm-minimal.sh:11]:
```suggestion
LOGGING_NS="${LOGGING_NS:-openshift-logging}"
```

File: hack/testing-olm/utils [hunk 2]
```diff
@@ -4,7 +4,7 @@ millisecond=1
 second=$(( 1000 * millisecond ))
 minute=$(( 60 * second ))
 
-LOGGING_NS="openshift-logging"
+LOGGING_NS=${LOGGING_NS:-openshift-logging}
```
Comment (vparfonov) [inline hack/testing-olm/utils:7]:
```suggestion
LOGGING_NS="${LOGGING_NS:-openshift-logging}"
```

File: test/e2e/telemetry/clometrics_test.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ TOKEN=`echo $(oc get secret $SECRET -n openshift-monitoring -o json | jq -r '.da
 THANOS_QUERIER_HOST=`oc get route thanos-querier -n openshift-monitoring -o json | jq -r '.spec.host'`
 echo $THANOS_QUERIER_HOST
 
-export NAMESPACE=openshift-logging
+export LOGGING_NS=${LOGGING_NS:-openshift-logging}
```
Comment (vparfonov) [inline test/e2e/telemetry/clometrics_test.sh:13]:
```suggestion
export LOGGING_NS="${LOGGING_NS:-openshift-logging}"
```

**Pull request merged**